sources2csr/helper_variables.py

```diff
@@ -36,32 +35,3 @@ def add_derived_values(subject_registry: CentralSubjectRegistry) -> CentralSubje
                 individual.age_first_diagnosis = relativedelta(first_diagnosis_date, individual.birth_date).years
 
     return subject_registry
-
-
-def add_ngs_data(subject_registry: CentralSubjectRegistry, ngs_set: Set[NGS]) -> CentralSubjectRegistry:
```

Comment:
maybe this file should be renamed back to `derived_values.py` then? The reason it was here was to have all helper variables in one place, but with ngs in a separate file, this name can be confusing

Reply:
I renamed it now.